test/one_time_password_test.rb

```diff
@@ -34,6 +38,17 @@ def test_counter_based_otp
     assert code != @member.otp_code(auto_increment: true)
   end
 
+  def test_counter_based_otp_active_record
```

Comment:
Assignment Branch Condition size for test_counter_based_otp_active_record is too high. [17.12/15]